tests/units/iop/test_parser/test_dois.py

```diff
@@ -0,0 +1,31 @@
+import xml.etree.ElementTree as ET
+
+from iop.parser import IOPParser
+from iop.xml_extractors import RequiredFieldNotFoundExtractionError
+from pytest import fixture, raises
+
+
+@fixture
+def parser():
+    return IOPParser()
+
+
+def test_happy_path(shared_datadir, parser):
+    with open(shared_datadir / "just_required_fields.xml") as file:
```

Comment:
please check the documentation how to read the files without using `open` https://pypi.org/project/pytest-datadir/